editor/src/clj/util/digestable.clj

```diff
@@ -232,7 +232,7 @@
    (sha1-hash object nil))
   (^String [object opts]
    (with-open [digest-output-stream (digest/make-digest-output-stream "SHA-1")
-               writer (OutputStreamWriter. digest-output-stream)]
+               writer (BufferedWriter. (OutputStreamWriter. digest-output-stream))]
```

Comment:
There was an allocation hotspot in `digest-tagged-bytes!` that writes bytes one by one. It created a lot of `java.nio.HeapCharBuffer` objects. Buffering helped.